src/domain/accounts/index.types.d.ts

```diff
@@ -62,10 +68,14 @@ type LimitsChecker = {
     walletVolume: TxVolume
   }): true | LimitsExceededError
 }
+
 interface IAccountsRepository {
   findById(accountId: AccountId): Promise<Account | RepositoryError>
   listByUserId(userId: UserId): Promise<Account[] | RepositoryError>
   findByWalletId(walletId: WalletId): Promise<Account | RepositoryError>
   findByWalletPublicId(walletPublicId: WalletPublicId): Promise<Account | RepositoryError>
+  findByPhone(phone: PhoneNumber): Promise<Account | RepositoryError>
```

Comment:
I think this can be deleted as you've already implemented the use case correctly:
```
user = userRepo.findByPhone(phone)
account = accounts.findById(user.defaultAccountId)
```

Reply:
you're right